tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.ng.html

```diff
@@ -0,0 +1,33 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<h3>Upload to TensorBoard.dev?</h3>
+<p>
+  TensorBoard.dev enables you to easily host, track, and share your ML
+  experiments with everyone. You can share a link to the uploaded TensorBoard in
+  papers, blog posts, and social media. This can showcase the results more
+  effectively and helps reproducibility.
+</p>
+<p>
+  To upload a logdir to TensorBoard.dev, run the command:
+</p>
+<div class="code">
```

Comment:
`code` and `pre` instead? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code

Reply:
I went with `<code>` and `<div>`.  I don't think we need or want the formatting of `<pre>`. If this becomes multiline then we might want to use `<pre>`.

Reply:
I think the `<code>` is supposed to be used when you want the code blocked to be an `display: inline` (kinda like the code formatting on this comment). You want it to be a `display: block`, you may need to use `pre`.

Reply:
I think `<pre>` is recommended not because it's a block element but because it allows for preformatted text, which can be very useful for code snippets.  I don't want the preformatted text at the moment.  But you're right that I want the block.  So I use a  `<div>` instead, since it's the standard general-purpose block element.  The `<code>` element here is still useful since it by default gives some code-link fonts/formatting.